### ISSUE TITLE
Confirm font style used for titles

### ISSUE THREAD

**Issue by anikey** [commitdev/commit.dev-proto#109]
Make sure the font used for the titles throughout the website is the same as design.
Design is using : 
Montserrat-Extrabold
32px font size
10% letter spacing

Current font weight looks a bit less bold

![font](https://user-images.githubusercontent.com/29130378/109178774-10e43100-773e-11eb-93bb-382cbc4957b5.png)

**Comment by adriandelisle** [comment]
@anikey Just had look through and the styles seem to match the css in figma. `line-height` is off by 1px, but I tested it out and it didn't make much of a difference.

Figma:
![Screen Shot 2021-02-25 at 4 49 46 PM](https://user-images.githubusercontent.com/10035832/109239060-7c072500-7789-11eb-90ce-8e6ab3ad5f44.png)
On https://launch.commit-staging.dev/:
![Screen Shot 2021-02-25 at 4 50 29 PM](https://user-images.githubusercontent.com/10035832/109241587-706a2d00-778e-11eb-88ec-664d8ff8ec11.png)
![Screen Shot 2021-02-25 at 4 51 56 PM](https://user-images.githubusercontent.com/10035832/109241590-7102c380-778e-11eb-961f-90ec8e3864e7.png)
![Screen Shot 2021-02-25 at 4 51 40 PM](https://user-images.githubusercontent.com/10035832/109241591-7102c380-778e-11eb-9570-f6df2078b3b2.png)

**Comment by anikey** [comment]
@adriandelisle interesting, I do see it follows the code Figma has, would it possible to test to see what happens if we have font-weight as 900?

**Comment by adriandelisle** [comment]
@anikey I did a quick change to my local build to see what it looks like:
![Screen Shot 2021-02-26 at 9 37 55 AM](https://user-images.githubusercontent.com/10035832/109335590-19f50100-7817-11eb-8148-e4ae216d8902.png)

If you want I can push up a change to https://launch.commit-staging.dev/.

**Comment by adriandelisle** [comment]
@anikey I found the actual issue, it was a bit deeper than I thought. When experimenting with font-weight: 900 I wasn't noticing any changes. I dived a bit deeper and we weren't actually requesting the 800 weight for `Montserrat` and it silently falls back to the largest size it has available (700). I'll push a fix.

(Left: proper font-weight requested, Right fall back to 700)
![Screen Shot 2021-02-26 at 9 55 47 AM](https://user-images.githubusercontent.com/10035832/109337075-1febe180-7819-11eb-9eb8-3aee696f7412.png)
